ProcessLib/BoundaryCondition/HCOpenBoundaryCondition.cpp

```diff
@@ -0,0 +1,68 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "HCOpenBoundaryCondition.h"
+
+#include "ProcessLib/Utils/ProcessUtils.h"
+
+namespace ProcessLib
+{
+std::unique_ptr<HCOpenBoundaryCondition> createHCOpenBoundaryCondition(
+    BaseLib::ConfigTree const& config, MeshLib::Mesh const& bc_mesh,
+    NumLib::LocalToGlobalIndexMap const& dof_table, int const variable_id,
+    int const component_id, unsigned const integration_order,
+    std::vector<std::unique_ptr<ParameterBase>> const& parameters,
+    unsigned const global_dim, Process const& process,
+    unsigned const shapefunction_order)
+{
+    DBUG(
+        "Constructing open boundary for Component Transport process from "
+        "config.");
+    //    //! \ogs_file_param{prj__processes__process__type}
+    //    config.checkConfigParameter("type", "ComponentTransport");
+    //    How can I check for process type == Component Transport? The method
+    //    above does not work.
```

Comment:
At this point you cannot. The process variables do not belong to a particular process. (A process uses some process variables, though.)

Reply:
Ok, so I drop it here.